wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/WMultiFileWidgetWebElementTest.java

```diff
@@ -0,0 +1,62 @@
+package com.github.bordertech.wcomponents.test;
+
+import com.github.bordertech.wcomponents.test.components.WMultiFileWidgetUI;
+import com.github.bordertech.wcomponents.test.selenium.ByButton;
+import com.github.bordertech.wcomponents.test.selenium.ByLabel;
+import com.github.bordertech.wcomponents.test.selenium.MultiBrowserRunner;
+import com.github.bordertech.wcomponents.test.selenium.SeleniumJettyTestCase;
+import com.github.bordertech.wcomponents.test.selenium.driver.SeleniumWComponentsWebDriver;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWButtonWebElement;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWMultiFileWidgetWebElement;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(MultiBrowserRunner.class)
+public class WMultiFileWidgetWebElementTest extends SeleniumJettyTestCase {
+
+
+	/**
+	 * Constructor...
+	 */
+	public WMultiFileWidgetWebElementTest() {
+		super(new WMultiFileWidgetUI());
+	}
+
+	@Test
+	@Ignore
+	public void testTextFieldFeatures() {
+		SeleniumWComponentsWebDriver driver = getDriver();
+
+		SeleniumWButtonWebElement validatingButton = driver.findWButton(new ByButton("Validate button", false, true));
+
+		validatingButton.click();
+
+		SeleniumWMultiFileWidgetWebElement wMultiFileWidgetWebElement =
+			driver.findWMultiFileWidget(new ByLabel("this is a textupload file widget field", false));
+
+		String filePath = getFilePath("attachment1.jpg");
+		wMultiFileWidgetWebElement.attachFile(filePath);
+
+		validatingButton.click();
+//		validatingButton.
+		assertEquals(1, wMultiFileWidgetWebElement.findFilesAttachedCount());
+
+
+		int i = 0;
+		i++;
+
+
+	}
+
+	private String getFilePath(String fileName) {
+		ClassLoader classLoader = getClass().getClassLoader();
+		File file = new File(classLoader.getResource(fileName).getFile());
+		return file.getAbsolutePath();
+	}
+
+}
```

Comment:
Again this is good for testing simple validation and the WFieldIndicatorWebElement but does not really test much of the WMultiFileWidgetWebElement which needs tests with multiple files, file constraints (size, file type) and the usual disabled and read-only states.